### PR TITLE
Dev example

### PR DESCRIPTION
**Pull request by alexvicegrab**

### CODE REVIEW

File: dev_example/README.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+#### CouchDB
+
+We start by installing the CouchDB database:
+
+    helm install stable/hlf-couchdb -n cdb-peer1 --namespace peers -f ../values/hlf-couchdb/cdb-peer1.yaml
+
+And check that it is running:
+
+    CDB_POD=$(kubectl get pods -n peers -l "app=hlf-couchdb,release=cdb-peer${NUM}" -o jsonpath="{.items[*].metadata.name}")
```
Comment (nicolapaoli) [inline dev_example/README.md]: `$NUM` is not defined. I guess we can keep `1` for this example

**Review by nicolapaoli — CHANGES_REQUESTED** [review]
All works, just a small thing to fix on commands.

File: dev_example/README.md (hunk 1 above)
Comment (alexvicegrab) [reply dev_example/README.md]: I will fix

Comment (alexvicegrab) [reply dev_example/README.md]: Fixed

**Review by nicolapaoli — APPROVED** [review]

**Pull request closed without merging**